src/components/comparison/memory-usage-panel/memory-usage-panel.tsx

```diff
@@ -126,19 +128,26 @@ export const MemoryUsagePanel = ({
             </StatContainer>
             {expandState === ExpandState.expanded && (
               <>
-                <StatContainer bottom={12}>
-                  <StatTitle>{`${tilesetStats.id.substring(
-                    0,
-                    37
-                  )}...`}</StatTitle>
-                  <LinkIcon
-                    fill={theme.colors.fontColor}
-                    style={{ cursor: "pointer" }}
-                    onClick={() => {
-                      navigator.clipboard.writeText(tilesetStats.id);
-                    }}
-                  />
-                </StatContainer>
+                {
+                  tilesetStats.id.split('<-tileset->').map(tilesetUrl => {
+                    const activeLayer = activeLayers.find(layer => layer.url === tilesetUrl);
+                    const activeLayerName = activeLayer?.name || `${tilesetUrl.substring(0, 37)}...`;
```

Comment:
what the magic number 37? If it is to fit to the panel width, use `text-overflow: ellipsis` css property

Reply:
Done